.editorconfig

```diff
@@ -0,0 +1,7 @@
+﻿[*.cs]
+
+# CA1822: Mark members as static
+dotnet_diagnostic.CA1822.severity = silent
+
+# CA1303: Do not pass literals as localized parameters
+dotnet_diagnostic.CA1303.severity = silent
```

Comment:
I was trying to do this just for the unit test project. Did I do that right?

Reply:
This will affect everything in the repository. You probably want to supress the warning with NoWarn in the csproj

Reply:
@Keboo I'm torn on 1303... As probably expected, when I removed these, some 1303 error showed up in the main project that I'm trying real hard not to turn off any of the analyzers on. But when I looked up the recommended fix, it seems... a bit excessive?

https://docs.microsoft.com/en-us/dotnet/api/system.resources.resourcemanager?view=netcore-3.1

Suggestions on following the approach recommend in the URL (creating a resource manager class backed by a .resources file)?

Reply:
That sems excessive unless we plan to support localization. I think simply disabling it for your project is the simplest option for now.

Reply:
@Keboo  Totally forgot to come back to this. Moved to NoWarns in each project and reverted .editorconfig settings.

Reply:
@Keboo  It looks like the NoWarn tags I added aren't getting recognized by the build server/pipeline? That seems strange... I'm gonna try a couple of different things, but may require your assistance if I can't get it figured out.

Reply:
@Keboo Details, details... I forgot to add the NoWarn for Release builds. :)